elastic/kibana
# [RAM] Add the Logs tab to Rules and Connectors UI

## Summary

Closes #137992

Adds a **Logs** tab to the Rules and Connectors UI. This tab populates itself with event logs from **every rule the current user has authorization to see.**

### Testing

1. Logged in as `elastic`, create some dummy rules of type **Index Threshold** and **Inventory**
2. Open the Logs tab and ensure all these rules populate it
3. Create a user role that does not have permissions to use Observability features, and then assign a user to it
4. Log in as this user. Ensure the Logs tab only shows entries from the **Index Threshold** rules
5. Create another space, and log into this space. Ensure the Logs tab shows nothing from the Default space
6. Create some rules on this second space. Ensure the Logs tab now shows log entries from rules on this second space, but the Default space doesn't show these entries when you switch back to it

<img width="1412" alt="Screen Shot 2022-08-15 at 3 52 48 PM" src="https://user-images.githubusercontent.com/1445834/184716384-7fccf0d3-5c6e-4f15-890a-7c882b9386b4.png">

### Checklist
